### PR TITLE
Add scripts for starting bench-tps

### PR DESCRIPTION
**Pull request by jstarry**

### CODE REVIEW

File: wrapper-bench-tps.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+exec > wrapper-bench-tps.log
+exec 2>&1
+
+tdsDir="$(pwd)"
+here=$(dirname "$0")
+netDir=$1
+clientId=$2
+txCount=$3
+
+cd $netDir
```
Comment (mvines) [inline wrapper-bench-tps.sh]: IMO, avoid the `cd` and just path to each net/ command explicitly.   Net can find itself

Comment (jstarry) [reply wrapper-bench-tps.sh]: Ah yeah, good point!

**Review by mvines — COMMENTED** [review]
The idea is that ramp-tps will always be running on some terminal right?  If it's interrupted (closed laptop lid without running in tmux), can we easily restart at the same state?   In this case the ssh sessions to the bench-tps clients will get closed and the bench-tps clients will stop

**Comment by jstarry** [comment]
Yeah that was the idea, I do have a mechanism for starting at roughly the same state but can certainly make it more robust.

File: wrapper-bench-tps.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+tdsDir="$(pwd)"
+netDir=$1
+clientId=$2
+txCount=$3
+
+eval "$("$netDir/gce.sh" info --eval)"
+clientIp="NET_CLIENT${clientId}_IP"
+
+"$netDir/scp.sh" $tdsDir/remote-bench-tps.sh solana@${!clientIp}:.
+"$netDir/ssh.sh" ${!clientIp} killall solana-bench-tps || true
```
Comment (mvines) [inline wrapper-bench-tps.sh]: just put this in `remote-bench-tps.sh`

**Comment by jstarry** [comment]
@mvines here's the mechanism for starting at a particular "TPS round" https://github.com/solana-labs/tour-de-sol/pull/28/ It has holes... hard to delegate stake awards idempotently for example

File: remote-bench-tps.sh [hunk 3]
```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
```
Comment (mvines) [inline remote-bench-tps.sh]: let's just merge this script into bench-tps.sh, the two scripts are close enough and 2 sucks

**Review by mvines — APPROVED** [review]

**Pull request merged**